cephci/utils/configure.py

```diff
@@ -76,3 +81,40 @@ def setup_ssh_keys(installer, nodes):
                 SSH_COPYID.format(SSH_ID_RSA_PUB, "root", node.hostname),
             ),
         )
+
+
+def accept_eula(node):
+    """
+    Accepts the eula for IBM Storage Ceph License
+    Args:
+        node(Node): Node object
+
+    """
+    cmd = "cat" if accept_eula else "touch"
+    cmd += " /usr/share/ibm-storage-ceph-license/accept"
+
+    Package(node).install("ibm-storage-ceph-license", env_vars=accept_eula)
+    return node.exec_command(sudo=True, cmd=cmd, check_ec=True)
+
+
+def exec_cephadm_preflight(installer, build_type, ceph_repo=None):
```

Comment:
where this function is called ?

Reply:
This will be called here : https://github.com/red-hat-storage/cephci/pull/2594/files#diff-be9b71a498f8f0d005762f98c531d6adf7bf0147b9dbce2a83dc31bab911acfd 

(prereq.py)  : 275

Reply:
Let it be part of this PR

Reply:
Also before calling preflight.yaml, need to have step to validate cephadm-package and ssh is setup.

Reply:
Added as part of this PR. SSH and ansible pkg install checks are done before calling the preflight execution